falco.yaml

```diff
@@ -98,8 +102,32 @@ pipeline:
       - runs: |
           make falco
           install -Dm755 ./userspace/falco/falco "${{targets.destdir}}"/usr/bin/falco
+      - uses: strip
 
-  - uses: strip
+subpackages:
+  - name: falco-no-driver
+    description: Cloud Native Runtime Security with modern eBPF support and without driver loader
+    pipeline:
+      - working-directory: build
+        pipeline:
+          - runs: |
+              cmake \
+                -DCMAKE_INSTALL_PREFIX=/usr \
+                -DCMAKE_INSTALL_LIBDIR=/usr/lib \
+                -DFALCO_ETC_DIR=/etc/falco \
+                -DCMAKE_BUILD_TYPE=Release \
+                -DUSE_BUNDLED_DEPS=Off \
+                -DMINIMAL_BUILD=On \
+                -DBUILD_DRIVER=Off \
+                -DBUILD_FALCO_MODERN_BPF=On \
+                -DBUILD_LIBSCAP_MODERN_BPF=Off \
+                -DLIBSCAP_INCLUDE_DIRS=/usr/lib/falco-libscap-modern-ebpf \
+                -DBUILD_BPF=Off \
```

Comment:
How this is different than `BUILD_FALCO_MODERN_BPF`?

Reply:
`BUILD_BPF` determines if the legacy (non-modern) eBPF driver should be built.

I'll put a link to the build docs for falco and the libs for reference, though not all of the flags are explicitly mentioned in the docs (e.g. `LIBSCAP_INCLUDE_DIRS`)